owncloudApp/src/main/java/com/owncloud/android/operations/common/OperationType.kt

```diff
@@ -0,0 +1,18 @@
+package com.owncloud.android.operations.common
+
+enum class OperationType {
```

Comment:
I think for all I have reviewed in these PRs, you can work enum, but do you know sealed classes in Kotlin? It's just a question...

Reply:
Yes but I thought enum class was enough for this case, is there any additional advantage of using sealed class instead of enum here?